src/services/orders.ts

```diff
@@ -12,15 +12,27 @@ import { getWeb3Wrapper } from './web3_wrapper';
 
 const logger = getLogger('Services::Orders');
 
-export const getAllOrders = (baseToken: Token, quoteToken: Token) => {
+const getAllOrders = async (baseToken: Token, quoteToken: Token, makerAddresses: string[] | null) => {
```

Comment:
Seems much more natural to have `makerAddresses?: string[]` - is there a reason reason to have the required param and `| null setup`?

Reply:
I did it this way because all of the fields in `MarketState` are defined definite as some time or `null`, but it looks like TS recognizes null as compatible with the `?` optional notation.

Reply:
It's this way because `makerAddress` and other fields in the `MarketState` interface are defined as some type or `null`. This pattern of defining optional params is used all across the code base unfortunately.